--------------------
Wed Jan 25 20:17:22 2017:  *Bad argument 2 to foreach
Expected: array Got: 0.
Object: /obj/handlers/armoury at line 541
Arguments were: (void)
Locals were: none.


(1) 'rehash' in /obj/handlers/armoury at line 361
(2) 'make_list' in /obj/handlers/armoury at line 154
(3) 'walk_directory' in /obj/handlers/armoury at line 541
--------------------
Wed Jan 25 20:17:27 2017:  *Bad argument 2 to foreach
Expected: array Got: 0.
Object: /obj/handlers/armoury at line 541
Arguments were: (void)
Locals were: none.


(1) 'rehash' in /obj/handlers/armoury at line 364
(2) 'make_list' in /obj/handlers/armoury at line 154
(3) 'walk_directory' in /obj/handlers/armoury at line 541
--------------------
Wed Jan 25 20:19:40 2017:  *Bad argument 1 to db_exec()
Expected: int Got: "Can't connect to local MySQL server through socket '/tmp/mysql.sock' (2 "No such file or directory")".
Object: /obj/handlers/map (/<driver>) in /(fake):0
Arguments were: (void)
Locals were: none.


(1) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
(2) 'check_next' in /obj/handlers/map at line 676
(3) 'check_map2' in /obj/handlers/map at line 899
(4) 'query_dest_dir' in /obj/handlers/map at line 444
(5) 'db_exec' in /secure/simul_efun (/secure/simul_efun/modified_efuns.c) at line 807
(6) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
--------------------
Wed Jan 25 20:21:44 2017:  *Bad argument 1 to db_exec()
Expected: int Got: "Can't connect to local MySQL server through socket '/tmp/mysql.sock' (2 "No such file or directory")".
Object: /obj/handlers/map (/<driver>) in /(fake):0
Arguments were: (void)
Locals were: none.


(1) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
(2) 'check_next' in /obj/handlers/map at line 676
(3) 'check_map2' in /obj/handlers/map at line 899
(4) 'query_dest_dir' in /obj/handlers/map at line 444
(5) 'db_exec' in /secure/simul_efun (/secure/simul_efun/modified_efuns.c) at line 807
(6) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
--------------------
Wed Jan 25 20:21:48 2017:  *Bad argument 1 to db_exec()
Expected: int Got: "Can't connect to local MySQL server through socket '/tmp/mysql.sock' (2 "No such file or directory")".
Object: /obj/handlers/map (/<driver>) in /(fake):0
Arguments were: (void)
Locals were: none.


(1) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
(2) 'check_next' in /obj/handlers/map at line 676
(3) 'check_map2' in /obj/handlers/map at line 899
(4) 'query_dest_dir' in /obj/handlers/map at line 444
(5) 'db_exec' in /secure/simul_efun (/secure/simul_efun/modified_efuns.c) at line 807
(6) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
--------------------
Wed Jan 25 20:21:52 2017:  *Bad argument 1 to db_exec()
Expected: int Got: "Can't connect to local MySQL server through socket '/tmp/mysql.sock' (2 "No such file or directory")".
Object: /obj/handlers/map (/<driver>) in /(fake):0
Arguments were: (void)
Locals were: none.


(1) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
(2) 'check_next' in /obj/handlers/map at line 676
(3) 'check_map2' in /obj/handlers/map at line 899
(4) 'query_dest_dir' in /obj/handlers/map at line 444
(5) 'db_exec' in /secure/simul_efun (/secure/simul_efun/modified_efuns.c) at line 807
(6) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
--------------------
Wed Jan 25 20:21:56 2017:  *Bad argument 1 to db_exec()
Expected: int Got: "Can't connect to local MySQL server through socket '/tmp/mysql.sock' (2 "No such file or directory")".
Object: /obj/handlers/map (/<driver>) in /(fake):0
Arguments were: (void)
Locals were: none.


(1) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
(2) 'check_next' in /obj/handlers/map at line 676
(3) 'check_map2' in /obj/handlers/map at line 899
(4) 'query_dest_dir' in /obj/handlers/map at line 444
(5) 'db_exec' in /secure/simul_efun (/secure/simul_efun/modified_efuns.c) at line 807
(6) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
--------------------
Wed Jan 25 20:22:00 2017:  *Bad argument 1 to db_exec()
Expected: int Got: "Can't connect to local MySQL server through socket '/tmp/mysql.sock' (2 "No such file or directory")".
Object: /obj/handlers/map (/<driver>) in /(fake):0
Arguments were: (void)
Locals were: none.


(1) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
(2) 'check_next' in /obj/handlers/map at line 676
(3) 'check_map2' in /obj/handlers/map at line 899
(4) 'query_dest_dir' in /obj/handlers/map at line 444
(5) 'db_exec' in /secure/simul_efun (/secure/simul_efun/modified_efuns.c) at line 807
(6) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
--------------------
Wed Jan 25 20:22:04 2017:  *Bad argument 1 to db_exec()
Expected: int Got: "Can't connect to local MySQL server through socket '/tmp/mysql.sock' (2 "No such file or directory")".
Object: /obj/handlers/map (/<driver>) in /(fake):0
Arguments were: (void)
Locals were: none.


(1) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
(2) 'check_next' in /obj/handlers/map at line 676
(3) 'check_map2' in /obj/handlers/map at line 899
(4) 'query_dest_dir' in /obj/handlers/map at line 444
(5) 'db_exec' in /secure/simul_efun (/secure/simul_efun/modified_efuns.c) at line 807
(6) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
--------------------
Wed Jan 25 20:22:08 2017:  *Bad argument 1 to db_exec()
Expected: int Got: "Can't connect to local MySQL server through socket '/tmp/mysql.sock' (2 "No such file or directory")".
Object: /obj/handlers/map (/<driver>) in /(fake):0
Arguments were: (void)
Locals were: none.


(1) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
(2) 'check_next' in /obj/handlers/map at line 676
(3) 'check_map2' in /obj/handlers/map at line 899
(4) 'query_dest_dir' in /obj/handlers/map at line 444
(5) 'db_exec' in /secure/simul_efun (/secure/simul_efun/modified_efuns.c) at line 807
(6) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
--------------------
Wed Jan 25 20:22:12 2017:  *Bad argument 1 to db_exec()
Expected: int Got: "Can't connect to local MySQL server through socket '/tmp/mysql.sock' (2 "No such file or directory")".
Object: /obj/handlers/map (/<driver>) in /(fake):0
Arguments were: (void)
Locals were: none.


(1) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
(2) 'check_next' in /obj/handlers/map at line 676
(3) 'check_map2' in /obj/handlers/map at line 899
(4) 'query_dest_dir' in /obj/handlers/map at line 444
(5) 'db_exec' in /secure/simul_efun (/secure/simul_efun/modified_efuns.c) at line 807
(6) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
--------------------
Wed Jan 25 20:22:16 2017:  *Bad argument 1 to db_exec()
Expected: int Got: "Can't connect to local MySQL server through socket '/tmp/mysql.sock' (2 "No such file or directory")".
Object: /obj/handlers/map (/<driver>) in /(fake):0
Arguments were: (void)
Locals were: none.


(1) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
(2) 'check_next' in /obj/handlers/map at line 676
(3) 'check_map2' in /obj/handlers/map at line 899
(4) 'query_dest_dir' in /obj/handlers/map at line 444
(5) 'db_exec' in /secure/simul_efun (/secure/simul_efun/modified_efuns.c) at line 807
(6) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
--------------------
Wed Jan 25 20:22:20 2017:  *Bad argument 1 to db_exec()
Expected: int Got: "Can't connect to local MySQL server through socket '/tmp/mysql.sock' (2 "No such file or directory")".
Object: /obj/handlers/map (/<driver>) in /(fake):0
Arguments were: (void)
Locals were: none.


(1) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
(2) 'check_next' in /obj/handlers/map at line 676
(3) 'check_map2' in /obj/handlers/map at line 899
(4) 'query_dest_dir' in /obj/handlers/map at line 444
(5) 'db_exec' in /secure/simul_efun (/secure/simul_efun/modified_efuns.c) at line 807
(6) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
--------------------
Wed Jan 25 20:22:24 2017:  *Bad argument 1 to db_exec()
Expected: int Got: "Can't connect to local MySQL server through socket '/tmp/mysql.sock' (2 "No such file or directory")".
Object: /obj/handlers/map (/<driver>) in /(fake):0
Arguments were: (void)
Locals were: none.


(1) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
(2) 'check_next' in /obj/handlers/map at line 676
(3) 'check_map2' in /obj/handlers/map at line 899
(4) 'query_dest_dir' in /obj/handlers/map at line 444
(5) 'db_exec' in /secure/simul_efun (/secure/simul_efun/modified_efuns.c) at line 807
(6) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
--------------------
Wed Jan 25 20:22:28 2017:  *Bad argument 1 to db_exec()
Expected: int Got: "Can't connect to local MySQL server through socket '/tmp/mysql.sock' (2 "No such file or directory")".
Object: /obj/handlers/map (/<driver>) in /(fake):0
Arguments were: (void)
Locals were: none.


(1) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
(2) 'check_next' in /obj/handlers/map at line 676
(3) 'check_map2' in /obj/handlers/map at line 899
(4) 'query_dest_dir' in /obj/handlers/map at line 444
(5) 'db_exec' in /secure/simul_efun (/secure/simul_efun/modified_efuns.c) at line 807
(6) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
--------------------
Wed Jan 25 20:22:32 2017:  *Bad argument 1 to db_exec()
Expected: int Got: "Can't connect to local MySQL server through socket '/tmp/mysql.sock' (2 "No such file or directory")".
Object: /obj/handlers/map (/<driver>) in /(fake):0
Arguments were: (void)
Locals were: none.


(1) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
(2) 'check_next' in /obj/handlers/map at line 676
(3) 'check_map2' in /obj/handlers/map at line 899
(4) 'query_dest_dir' in /obj/handlers/map at line 444
(5) 'db_exec' in /secure/simul_efun (/secure/simul_efun/modified_efuns.c) at line 807
(6) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
--------------------
Wed Jan 25 20:22:36 2017:  *Bad argument 1 to db_exec()
Expected: int Got: "Can't connect to local MySQL server through socket '/tmp/mysql.sock' (2 "No such file or directory")".
Object: /obj/handlers/map (/<driver>) in /(fake):0
Arguments were: (void)
Locals were: none.


(1) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
(2) 'check_next' in /obj/handlers/map at line 676
(3) 'check_map2' in /obj/handlers/map at line 899
(4) 'query_dest_dir' in /obj/handlers/map at line 444
(5) 'db_exec' in /secure/simul_efun (/secure/simul_efun/modified_efuns.c) at line 807
(6) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
--------------------
Wed Jan 25 20:22:40 2017:  *Bad argument 1 to db_exec()
Expected: int Got: "Can't connect to local MySQL server through socket '/tmp/mysql.sock' (2 "No such file or directory")".
Object: /obj/handlers/map (/<driver>) in /(fake):0
Arguments were: (void)
Locals were: none.


(1) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
(2) 'check_next' in /obj/handlers/map at line 676
(3) 'check_map2' in /obj/handlers/map at line 899
(4) 'query_dest_dir' in /obj/handlers/map at line 444
(5) 'db_exec' in /secure/simul_efun (/secure/simul_efun/modified_efuns.c) at line 807
(6) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
--------------------
Wed Jan 25 20:22:44 2017:  *Bad argument 1 to db_exec()
Expected: int Got: "Can't connect to local MySQL server through socket '/tmp/mysql.sock' (2 "No such file or directory")".
Object: /obj/handlers/map (/<driver>) in /(fake):0
Arguments were: (void)
Locals were: none.


(1) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
(2) 'check_next' in /obj/handlers/map at line 676
(3) 'check_map2' in /obj/handlers/map at line 899
(4) 'query_dest_dir' in /obj/handlers/map at line 444
(5) 'db_exec' in /secure/simul_efun (/secure/simul_efun/modified_efuns.c) at line 807
(6) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
--------------------
Wed Jan 25 20:22:48 2017:  *Bad argument 1 to db_exec()
Expected: int Got: "Can't connect to local MySQL server through socket '/tmp/mysql.sock' (2 "No such file or directory")".
Object: /obj/handlers/map (/<driver>) in /(fake):0
Arguments were: (void)
Locals were: none.


(1) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
(2) 'check_next' in /obj/handlers/map at line 676
(3) 'check_map2' in /obj/handlers/map at line 899
(4) 'query_dest_dir' in /obj/handlers/map at line 444
(5) 'db_exec' in /secure/simul_efun (/secure/simul_efun/modified_efuns.c) at line 807
(6) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
--------------------
Wed Jan 25 20:22:52 2017:  *Bad argument 1 to db_exec()
Expected: int Got: "Can't connect to local MySQL server through socket '/tmp/mysql.sock' (2 "No such file or directory")".
Object: /obj/handlers/map (/<driver>) in /(fake):0
Arguments were: (void)
Locals were: none.


(1) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
(2) 'check_next' in /obj/handlers/map at line 676
(3) 'check_map2' in /obj/handlers/map at line 899
(4) 'query_dest_dir' in /obj/handlers/map at line 444
(5) 'db_exec' in /secure/simul_efun (/secure/simul_efun/modified_efuns.c) at line 807
(6) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
--------------------
Wed Jan 25 20:22:56 2017:  *Bad argument 1 to db_exec()
Expected: int Got: "Can't connect to local MySQL server through socket '/tmp/mysql.sock' (2 "No such file or directory")".
Object: /obj/handlers/map (/<driver>) in /(fake):0
Arguments were: (void)
Locals were: none.


(1) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
(2) 'check_next' in /obj/handlers/map at line 676
(3) 'check_map2' in /obj/handlers/map at line 899
(4) 'query_dest_dir' in /obj/handlers/map at line 444
(5) 'db_exec' in /secure/simul_efun (/secure/simul_efun/modified_efuns.c) at line 807
(6) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
--------------------
Wed Jan 25 20:23:00 2017:  *Bad argument 1 to db_exec()
Expected: int Got: "Can't connect to local MySQL server through socket '/tmp/mysql.sock' (2 "No such file or directory")".
Object: /obj/handlers/map (/<driver>) in /(fake):0
Arguments were: (void)
Locals were: none.


(1) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
(2) 'check_next' in /obj/handlers/map at line 676
(3) 'check_map2' in /obj/handlers/map at line 899
(4) 'query_dest_dir' in /obj/handlers/map at line 444
(5) 'db_exec' in /secure/simul_efun (/secure/simul_efun/modified_efuns.c) at line 807
(6) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
--------------------
Wed Jan 25 20:23:04 2017:  *Bad argument 1 to db_exec()
Expected: int Got: "Can't connect to local MySQL server through socket '/tmp/mysql.sock' (2 "No such file or directory")".
Object: /obj/handlers/map (/<driver>) in /(fake):0
Arguments were: (void)
Locals were: none.


(1) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
(2) 'check_next' in /obj/handlers/map at line 676
(3) 'check_map2' in /obj/handlers/map at line 899
(4) 'query_dest_dir' in /obj/handlers/map at line 444
(5) 'db_exec' in /secure/simul_efun (/secure/simul_efun/modified_efuns.c) at line 807
(6) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
--------------------
Wed Jan 25 20:23:08 2017:  *Bad argument 1 to db_exec()
Expected: int Got: "Can't connect to local MySQL server through socket '/tmp/mysql.sock' (2 "No such file or directory")".
Object: /obj/handlers/map (/<driver>) in /(fake):0
Arguments were: (void)
Locals were: none.


(1) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
(2) 'check_next' in /obj/handlers/map at line 676
(3) 'check_map2' in /obj/handlers/map at line 899
(4) 'query_dest_dir' in /obj/handlers/map at line 444
(5) 'db_exec' in /secure/simul_efun (/secure/simul_efun/modified_efuns.c) at line 807
(6) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
--------------------
Wed Jan 25 20:36:00 2017:  *Bad argument 2 to foreach
Expected: array Got: 0.
Object: /obj/handlers/armoury at line 541
Arguments were: (void)
Locals were: none.


(1) 'rehash' in /obj/handlers/armoury at line 361
(2) 'make_list' in /obj/handlers/armoury at line 154
(3) 'walk_directory' in /obj/handlers/armoury at line 541
--------------------
Wed Jan 25 20:36:05 2017:  *Bad argument 2 to foreach
Expected: array Got: 0.
Object: /obj/handlers/armoury at line 541
Arguments were: (void)
Locals were: none.


(1) 'rehash' in /obj/handlers/armoury at line 364
(2) 'make_list' in /obj/handlers/armoury at line 154
(3) 'walk_directory' in /obj/handlers/armoury at line 541
--------------------
Wed Jan 25 20:39:33 2017:  *Bad argument 1 to db_exec()
Expected: int Got: "Can't connect to local MySQL server through socket '/tmp/mysql.sock' (2 "No such file or directory")".
Object: /obj/handlers/map (/<driver>) in /(fake):0
Arguments were: (void)
Locals were: none.


(1) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
(2) 'check_next' in /obj/handlers/map at line 676
(3) 'check_map2' in /obj/handlers/map at line 899
(4) 'query_dest_dir' in /obj/handlers/map at line 444
(5) 'db_exec' in /secure/simul_efun (/secure/simul_efun/modified_efuns.c) at line 807
(6) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
--------------------
Wed Jan 25 20:41:37 2017:  *Bad argument 1 to db_exec()
Expected: int Got: "Can't connect to local MySQL server through socket '/tmp/mysql.sock' (2 "No such file or directory")".
Object: /obj/handlers/map (/<driver>) in /(fake):0
Arguments were: (void)
Locals were: none.


(1) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
(2) 'check_next' in /obj/handlers/map at line 676
(3) 'check_map2' in /obj/handlers/map at line 899
(4) 'query_dest_dir' in /obj/handlers/map at line 444
(5) 'db_exec' in /secure/simul_efun (/secure/simul_efun/modified_efuns.c) at line 807
(6) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
--------------------
Wed Jan 25 20:41:41 2017:  *Bad argument 1 to db_exec()
Expected: int Got: "Can't connect to local MySQL server through socket '/tmp/mysql.sock' (2 "No such file or directory")".
Object: /obj/handlers/map (/<driver>) in /(fake):0
Arguments were: (void)
Locals were: none.


(1) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
(2) 'check_next' in /obj/handlers/map at line 676
(3) 'check_map2' in /obj/handlers/map at line 899
(4) 'query_dest_dir' in /obj/handlers/map at line 444
(5) 'db_exec' in /secure/simul_efun (/secure/simul_efun/modified_efuns.c) at line 807
(6) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
--------------------
Wed Jan 25 20:41:41 2017:  *Inherited file '/std/outside' does not exist!
Object: /std/room/basic/door#22 at line 315
Arguments were: (void)
Locals were: none.


(1) 'process_input' in /global/lord#6 at line 85
(2) 'process_input' in /global/lord#6 (/global/wiz_file_comm.c) at line 510
(3) 'process_input' in /global/lord#6 (/global/psoul.c) at line 519
(4) '_process_input' in /global/lord#6 (/global/psoul.c) at line 589
(5) 'drunk_check' in /global/lord#6 (/global/psoul.c) at line 491
(6) 'command' in /global/lord#6 (/global/wiz_file_comm.c) at line 514
(7) 'command' in /global/lord#6 (/global/psoul.c) at line 615
(8) '_process_input' in /global/lord#6 (/global/psoul.c) at line 590
(9) 'exit_command' in /global/lord#6 (/std/living/living.c) at line 776
(10) 'exit_move' in /obj/handlers/room_handler at line 430
(11) 'force_other' in /std/room/basic/door#22 at line 315
--------------------
Wed Jan 25 20:41:45 2017:  *Bad argument 1 to db_exec()
Expected: int Got: "Can't connect to local MySQL server through socket '/tmp/mysql.sock' (2 "No such file or directory")".
Object: /obj/handlers/map (/<driver>) in /(fake):0
Arguments were: (void)
Locals were: none.


(1) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
(2) 'check_next' in /obj/handlers/map at line 676
(3) 'check_map2' in /obj/handlers/map at line 899
(4) 'query_dest_dir' in /obj/handlers/map at line 444
(5) 'db_exec' in /secure/simul_efun (/secure/simul_efun/modified_efuns.c) at line 807
(6) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
--------------------
Wed Jan 25 20:41:49 2017:  *Bad argument 1 to db_exec()
Expected: int Got: "Can't connect to local MySQL server through socket '/tmp/mysql.sock' (2 "No such file or directory")".
Object: /obj/handlers/map (/<driver>) in /(fake):0
Arguments were: (void)
Locals were: none.


(1) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
(2) 'check_next' in /obj/handlers/map at line 676
(3) 'check_map2' in /obj/handlers/map at line 899
(4) 'query_dest_dir' in /obj/handlers/map at line 444
(5) 'db_exec' in /secure/simul_efun (/secure/simul_efun/modified_efuns.c) at line 807
(6) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
--------------------
Wed Jan 25 20:42:53 2017:  *Bad argument 1 to db_exec()
Expected: int Got: "Can't connect to local MySQL server through socket '/tmp/mysql.sock' (2 "No such file or directory")".
Object: /obj/handlers/map (/<driver>) in /(fake):0
Arguments were: (void)
Locals were: none.


(1) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
(2) 'check_next' in /obj/handlers/map at line 676
(3) 'check_map2' in /obj/handlers/map at line 899
(4) 'query_dest_dir' in /obj/handlers/map at line 444
(5) 'db_exec' in /secure/simul_efun (/secure/simul_efun/modified_efuns.c) at line 807
(6) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
--------------------
Wed Jan 25 20:42:57 2017:  *Bad argument 1 to db_exec()
Expected: int Got: "Can't connect to local MySQL server through socket '/tmp/mysql.sock' (2 "No such file or directory")".
Object: /obj/handlers/map (/<driver>) in /(fake):0
Arguments were: (void)
Locals were: none.


(1) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
(2) 'check_next' in /obj/handlers/map at line 676
(3) 'check_map2' in /obj/handlers/map at line 899
(4) 'query_dest_dir' in /obj/handlers/map at line 444
(5) 'db_exec' in /secure/simul_efun (/secure/simul_efun/modified_efuns.c) at line 807
(6) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
--------------------
Wed Jan 25 20:43:01 2017:  *Bad argument 1 to db_exec()
Expected: int Got: "Can't connect to local MySQL server through socket '/tmp/mysql.sock' (2 "No such file or directory")".
Object: /obj/handlers/map (/<driver>) in /(fake):0
Arguments were: (void)
Locals were: none.


(1) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
(2) 'check_next' in /obj/handlers/map at line 676
(3) 'check_map2' in /obj/handlers/map at line 899
(4) 'query_dest_dir' in /obj/handlers/map at line 444
(5) 'db_exec' in /secure/simul_efun (/secure/simul_efun/modified_efuns.c) at line 807
(6) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
--------------------
Wed Jan 25 20:43:05 2017:  *Bad argument 1 to db_exec()
Expected: int Got: "Can't connect to local MySQL server through socket '/tmp/mysql.sock' (2 "No such file or directory")".
Object: /obj/handlers/map (/<driver>) in /(fake):0
Arguments were: (void)
Locals were: none.


(1) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
(2) 'check_next' in /obj/handlers/map at line 676
(3) 'check_map2' in /obj/handlers/map at line 899
(4) 'query_dest_dir' in /obj/handlers/map at line 444
(5) 'db_exec' in /secure/simul_efun (/secure/simul_efun/modified_efuns.c) at line 807
(6) '<fake>' in /obj/handlers/map (/<driver>) in /(fake):0
--------------------
Wed Jan 25 20:43:48 2017:  **Wrong permissions for opening file /save/newsrc/e/eron.o for overwrite.
"No such file or directory"
Object: /secure/master at line 362
Arguments were: (void)
Locals were: none.


(1) 'flush_newsrc' in /obj/handlers/board_handler at line 1330
(2) 'unguarded' in /secure/simul_efun (/secure/simul_efun/unguarded.c) at line 43
(3) 'apply_unguarded' in /secure/master at line 362
--------------------
Wed Jan 25 21:02:33 2017:  *Array index out of bounds.
Object: /obj/misc/torch#13 (/std/basic/id_match.c) at line 64
Arguments were: (void)
Locals were: none.


(1) 'process_input' in /global/lord#6 at line 85
(2) 'process_input' in /global/lord#6 (/global/wiz_file_comm.c) at line 510
(3) 'process_input' in /global/lord#6 (/global/psoul.c) at line 519
(4) '_process_input' in /global/lord#6 (/global/psoul.c) at line 589
(5) 'drunk_check' in /global/lord#6 (/global/psoul.c) at line 491
(6) 'command' in /global/lord#6 (/global/wiz_file_comm.c) at line 514
(7) 'command' in /global/lord#6 (/global/psoul.c) at line 615
(8) '_process_input' in /global/lord#6 (/global/psoul.c) at line 591
(9) 'cmdAll' in /global/lord#6 (/global/command.c) at line 65
(10) 'cmd' in /secure/cmds/creator/ca_ll at line 58
(11) 'wiz_present' in /global/wiz_present at line 216
(12) 'wzpresent2' in /global/wiz_present at line 78
(13) 'match_objects_in_environments' in /secure/simul_efun (/secure/simul_efun/obj_parser.c) at line 683
(14) 'match_object_in_array' in /secure/simul_efun (/secure/simul_efun/obj_parser.c) at line 454
(15) 'parse_match_object' in /obj/misc/torch#13 (/std/basic/id_match.c) at line 200
(16) 'is_matching_object' in /obj/misc/torch#13 (/std/basic/id_match.c) at line 64
